lib/syslogger.rb

```diff
@@ -156,6 +157,14 @@ def clear_tags!
 
   protected
 
+  # stolen from ruby/logger -- formatter expects string representation of severity
+  # Severity label for logging (max 5 chars).
+  SEV_LABEL = %w(DEBUG INFO WARN ERROR FATAL ANY)
```

Comment:
There's a coupling between the order of these and the integer values of the `Logger::` constants (since the integer value is used to lookup the label).  Given that the `MAPPING` hash uses the log levels in numerical order I think it would be easier to see the coupling and see that the order is defined properly if you move the definition of `SEV_LABEL` to just after the `MAPPING` definition.